tock/utilization/views.py

```diff
@@ -50,3 +57,82 @@ def get_context_data(self, **kwargs):
         context.update({'org_totals': org_billing_context()}
         )
         return context
+
+
+def _plot_utilization(dates, billable, nonbillable):
+    """Make a stacked area plot of billable and nonbillable hours.
+
+    dates, billable, and nonbillable should be sequences with the same length
+    """
+    fig = go.Figure()
+
+    fig.add_trace(go.Scatter(
+        x=dates,
+        y=billable,
+        line_shape="hv",
+        mode="lines+markers",
+        stackgroup="one",
+        name="billable",
+    ))
+    fig.add_trace(go.Scatter(
+        x=dates,
+        y=nonbillable,
+        line_shape="hv",
+        mode="lines+markers",
+        stackgroup="one",
+        name="nonbillable",
+    ))
+
+    fig.update_layout(
+        # autosize=False,
+        # width=900,
+        # height=500,
+        xaxis=dict(autorange=True),
+        yaxis=dict(autorange=True),
+        xaxis_title="Reporting Period Start Date",
+        yaxis_title="Hours",
+        title="Total Hours recorded vs. Time",
+    )
+
+    plot_div = plot(fig, output_type='div', include_plotlyjs=False)
+    return plot_div
+
+
+def _utilization_data(start_date, end_date):
+    Timecard = apps.get_model("hours", "Timecard")
+    billable_filter = Q(timecardobjects__project__accounting_code__billable=True)
+    data = (Timecard.objects.filter(reporting_period__start_date__gte=start_date,
```

Comment:
Opportunity here to limit this to submitted timecards by filtering on `Timecard.submitted`